arboretum/permissions/checks/test_org_permissions.py

```diff
@@ -0,0 +1,109 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Repository organization permissions check."""
+
+from compliance.check import ComplianceCheck
+from compliance.evidence import DAY, ReportEvidence, evidences
+from compliance.utils.data_parse import get_sha256_hash
+
+
+class OrgPermissionsCheck(ComplianceCheck):
+    """Monitor repository organization permissions."""
+
+    @property
+    def title(self):
+        """
+        Return the title of the checks.
+
+        :returns: the title of the checks
+        """
+        return 'Report on Repository Organization Permissions'
+
+    @classmethod
+    def setUpClass(cls):
+        """Initialize the check object with configuration settings."""
+        cls.config.add_evidences(
+            [
+                ReportEvidence(
+                    'org_permissions.md',
+                    'permissions',
+                    DAY,
+                    'Repository Permissions.'
+                )
+            ]
+        )
+        return cls
+
+    def test_org_permissions(self):
+        """Check the access to organization repositories."""
+        orgs = self.config.get('org.permissions.org_integrity.orgs')
+        for org in orgs:
+            host = org['url'].rsplit('/', 1)
```

Comment:
This is wrong.  A split returns a list.  Any comparison done on L55 and L57 will never work.  You're basically introducing a dormant bug that will wake up if we ever add support for GL and BB.

Reply:
Done